crates/re_log_types/src/path/parse_path.rs

```diff
@@ -154,44 +146,41 @@ impl EntityPath {
     /// Parses anything that `ent_path.to_string()` outputs.
     ///
     /// For a forgiving parse that accepts anything, use [`Self::parse_forgiving`].
-    pub fn parse_strict(s: &str) -> Result<Self, PathParseError> {
+    pub fn parse_strict(input: &str) -> Result<Self, PathParseError> {
         let DataPath {
             entity_path,
             instance_key,
             component_name,
-        } = DataPath::from_str(s)?;
+        } = DataPath::from_str(input)?;
 
         if let Some(instance_key) = instance_key {
             return Err(PathParseError::UnexpectedInstanceKey(instance_key));
         }
         if let Some(component_name) = component_name {
             return Err(PathParseError::UnexpectedComponentName(component_name));
         }
+
+        let normalized = entity_path.to_string();
+        if normalized != input {
+            re_log::warn_once!("The entity path '{input}' that was not in the normalized form. It will be interpreted as '{normalized}'. See https://www.rerun.io/docs/concepts/entity-path for more");
```

Comment:
```suggestion
            re_log::warn_once!("The entity path '{input}' was not in the normalized form. It will be interpreted as '{normalized}'. See https://www.rerun.io/docs/concepts/entity-path for more");
```